packages/alice/lib/ui/common/alice_dialog.dart

```diff
@@ -17,7 +17,7 @@ class AliceGeneralDialog {
   }) =>
       showDialog<void>(
         context: context,
-        builder: (BuildContext buildContext) {
+        builder: (BuildContext context) {
```

Comment:
Without that, the wrong context is pop (the parent one) and instead of dialog pop we got root navigation pop...